### PR TITLE
Android.TextEdit: fix TextEditRenderer alignment issues

### PR DESCRIPTION
**Pull request by nekronos**
Text alignment always broke the first time drawing due to the internal text alignment state in TextView not being valid. The previous workaround was to align the text ourselves with a grid. This aligned the text correct as long as it did not overflow its layout bounds. By reading the source code for TextView I figured out what to do to make the internal state update appropriately. See the code for more details.

This PR contains:
- [x] Changelog
- [ ] Documentation
- [x] Tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@
 
 ## WrapPanel
 - Added possibility to use `RowAlignment` to align the elements in the major direction of the `WrapPanel` as well as in the minor.
+# TextInput
+- Fixed issue on Android causing text to align incorrectly if being scrolled and unfocused.
+
+## WrapPanel
```
Comment (mortoray) [inline CHANGELOG.md]: This doesn't belong, snuck in somehow.

File: Source/Fuse.Controls.Primitives/TextControls/MobileTextEdit.uno [hunk 2]
```diff
@@ -317,51 +327,43 @@ namespace Fuse.Controls
...
+			target.setLayoutParams(new android.widget.FrameLayout.LayoutParams(width, height));
+
+			if (android.os.Build.VERSION.SDK_INT >= 17)
+				target.setTextAlignment(android.view.View.TEXT_ALIGNMENT_GRAVITY);
+
+			target.setGravity(source.getGravity());
+
+			target.setHorizontallyScrolling(!isMultiline);
+
+			if (firstFrame)
```
Comment (mortoray) [inline Source/Fuse.Controls.Primitives/TextControls/MobileTextEdit.uno]: Based on the comment wouldn't it mean you need to set `firstFrame = true` whenever one of those properites changes as well?

Comment (nekronos) [inline Source/Fuse.Controls.Primitives/TextControls/MobileTextEdit.uno]: Yep you are right. Fixed the code and test for changes in textalignment

**Comment by nekronos** [comment]
@mortoray bump

**Pull request merged**